hw/net/rocker/rocker_p4_l2l3_features.h

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2013-present Barefoot Networks, Inc. 
```

Comment:
I had the same question. I think it has to be a specific year. I'll work
internally to get it fixed.

On Sun, Apr 19, 2015 at 10:03 PM, Scott Feldman <notifications@github.com>
wrote:

> In hw/net/rocker/rocker_p4_l2l3_features.h
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28664265>:
>
> > @@ -0,0 +1,21 @@
> > +/*
> > +Copyright 2013-present Barefoot Networks, Inc.
>
> Not sure about open-ended copyrights. Past and current dates OK, but
> future? I'm not sure...might raise a flag.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28664265>.
>